lib/pp.ml

```diff
@@ -351,6 +351,11 @@ let pr_nth n =
   in
   int n ++ str s
 
+let pluralize word l : std_ppcmds =
+  if List.length l > 1 || List.length l == 0
```

Comment:
Fair enough, just didn't think about it.

This does make me nervous from an i18n/l10n perspective (this really doesn't work across languages, or even for all English words), not that Coq is getting translated any time soon.